checker/check_runner.go

```diff
@@ -98,8 +98,14 @@ func (r *Runner) Run(ctx context.Context, c Check) CheckResult {
 
 	ctx, err := tag.New(ctx, tag.Upsert(stats.CheckName, r.CheckName))
 	if err != nil {
-		panic(err)
+		r.CheckRequest.Dlogger.Debug(&LogMessage{Text: fmt.Sprintf("tag.New: %v", err)})
 	}
+
+	ctx, err = tag.New(ctx, tag.Upsert(stats.RepoHost, r.CheckRequest.Repo.Host()))
+	if err != nil {
+		r.CheckRequest.Dlogger.Debug(&LogMessage{Text: fmt.Sprintf("tag.New: %v", err)})
```

Comment:
I'm not sure the cron will see debug statements, do we just want to return these as error like we do in `logStats` above?

Reply:
I could change that to a warning, but I don't see logging failure as a reason the entire checkrun should fail.

Reply:
can we log in a way that doesn't associate it with the the check request? it's not really a detail of the check either